### DISCUSSION TITLE
Handeling DATE type

### DISCUSSION THREAD

**Discussion by ghost** [oracle/python-oracledb#55] [Q&A] [answered]
Hi team,

I Am trying to fetch data and write it to a CSV file which is completely fine.
But when there is a **date** type column, am getting with **date and time** which is by default package handles them as datetime.datetime.

So, I would like to get it as a date only. Is it possible to handle with **[connection.outputtypehandler](https://python-oracledb.readthedocs.io/en/latest/user_guide/sql_execution.html#:~:text=connection.outputtypehandler)** as **date** instead of datetime.datetime like NUMBER to DECIMAL in the example?

Thanks in advance.

**Comment by ghost** [comment, marked as answer]
This was resolved using **Outupthandler & Outconverter**
def datetime_to_date(value):
return datetime.date.strftime(value,"%Y-%m-%d")

def output_type_handler(cursor, name, default_type, size, precision, scale):
    if default_type == odb.DB_TYPE_DATE:
        return cursor.var(datetime.date, arraysize=cursor.arraysize,outconverter=datetime_to_date)

**Reply by anthony-tuininga** [reply]
Glad to hear that it was resolved! For the benefit of others, though, can you supply a brief answer to your question? I presume it involved the use of an output type handler as you suggested in your question?
